language/diem-framework/modules/doc/DiemConsensusConfig.md

```diff
@@ -15,7 +15,6 @@ DiemConfig, and may be updated by Diem root.
 
 
 <pre><code><b>use</b> <a href="DiemConfig.md#0x1_DiemConfig">0x1::DiemConfig</a>;
-<b>use</b> <a href="DiemTimestamp.md#0x1_DiemTimestamp">0x1::DiemTimestamp</a>;
```

Comment:
Silly question, but I'm guessing these are auto-generated?

Reply:
yeah, it's generated by the diem-framework binary